README.md

```diff
@@ -1,70 +1,35 @@
-# Getting Started with Create React App
+# Weather App
```

Comment:
This is a nice basic README, but it is lacking in a few things. Some things that would make it much better:
- screenshots of the app in the browser (mobile and desktop screen sizes especially if the app is responsive),
- A bit more info in the description, ie does it work for all cities in all countries?
- What would you do if you had more time?
- your name!